dcos_prerequisites_script

```diff
@@ -4,6 +4,7 @@
 # perl tar xz unzip curl bind-utils net-tools ipset libtool-ltdl rsync nfs-utils
 # sets up filesystem mounts
 # Installs and configures Docker
+# It is used in the Advanced Insall Quick Start blog post located at https://mesosphere.com/blog/dcos-advanced-installation-guide/
```

Comment:
Typo on the word "Install"

While you're in here, style-wise for code, you also probably want to put the URL on a new comment line so it doesn't trail off the edge of the screen or wrap awkwardly when people browse the code.

Reply:
+1 ^